pkgs/applications/audio/alsa-scarlett-gui/default.nix

```diff
@@ -25,8 +25,8 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ pkg-config wrapGAppsHook4 ];
   buildInputs = [ gtk4 alsa-lib ];
   postInstall = ''
-      substituteInPlace $out/share/applications/vu.b4.${pname}.desktop \
-        --replace "Exec=/bin/alsa-scarlett-gui" "Exec=$out/bin/${pname}"
+    substituteInPlace $out/share/applications/vu.b4.${pname}.desktop \
```

Comment:
```suggestion
    substituteInPlace $out/share/applications/vu.b4.alsa-scarlett-gui.desktop \
```

Reply:
Wasn't sure if you only meant in search/replace of everywhere - thank ✌🏻